src/index.ts

```diff
@@ -16,6 +18,31 @@ const store = new TimeStore({
   keepEventLoopAlive: false
 });
 
+const targets: any[] = [
```

Comment:
```suggestion
const targets: TransportTargetOptions[] = [
```
We can use `TransportTargetOptions` but maybe the `any` is intended ?